src/dispatch/status_report/flows.py

```diff
@@ -46,9 +46,10 @@ def save_status_report(
 
     event_service.log(
         db_session=db_session,
-        source="Dispatch Core App",
-        description=f"New status report created by {participant.individual.name}",
+        source="Incident Participant",
+        description=f"{participant.individual.name} created a new status report. Conditions: {conditions}; Actions: {actions}; Needs: {needs}",
```

Comment:
This might eventually stress the display of the timeline. I wonder if we could add "metadata" to the event table and store any structured data for the event there?

Reply:
Agree. I'll find another way to display large piece of text and add the column to the model in a separate PR.